src/styles/style.css

```diff
@@ -1,6 +1,7 @@
 body {
 	--invest-green: #148F68;
 	background-color: rgba(0,0,0,0);
+	font-size: 16px;
```

Comment:
Should this be in `rem` to allow it to scale?

Reply:
I would think 1 rem so it starts at the default size

Reply:
So the only reason I added this was because I was asking myself, "what controls the default font size?". I learned the browser has a default set at 16px, but thought I would define it here in case I'm asking that question again in 6 months.

I guess it could be `1rem`, but it's functionally no different unless chromium changes it's underlying default size. For all other rules except this "root" font-size rule, I agree we should use `rem`. 

The font is still scalable by the user this way, and the browser still remembers if you increased/decreased the size last time you had it open.

Reply:
Okay, sounds good. I think if we do want to set the default font size, the most correct way to do that would be with [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root), because `rem`s are based off the root element, and `<body />` is not the root element. But in practice it won't make a difference - they should always be the same.